### PR TITLE
Add some visual improvements

### PR DESCRIPTION
**Pull request by dragomano**

### CODE REVIEW

**Comment by albertlast** [comment]
Can you please add some pic?

**Comment by dragomano** [comment]
> Can you please add some pic?

Indeed.

Before:
browse packages - old view
After:
browse packages - new view

Before:
browse packages - id column - old view
After:
browse packages - id column - new view

Before:
manage bans - old view
After:
manage bans - new view

File: Sources/ManageBans.php [hunk 1]
```diff
@@ -268,6 +268,7 @@ function BanList()
 		'additional_rows' => array(
 			array(
 				'position' => 'top_of_list',
+				'class' => 'floatright',
```
Comment (SychO9) [inline Sources/ManageBans.php]: I would suggest removing the added `floatright` classes and editing the CSS code instead by adding after `.additional_rows`

```css
.additional_row::after {
    content: "";
    display: block;
    clear: both;
}
```

that way all elements of that type will be fixed and not only the ones where you had added the `floatright` class

File: Themes/default/index.template.php [hunk 2]
```diff
@@ -325,7 +325,7 @@ function template_body_above()
 	echo '
 	<div id="header">
 		<h1 class="forumtitle">
-			<a id="top" href="', $scripturl, '">', empty($context['header_logo_url_html_safe']) ? $context['forum_name_html_safe'] : '<img src="' . $context['header_logo_url_html_safe'] . '" alt="' . $context['forum_name_html_safe'] . '">', '</a>
+			', !empty($context['current_action']) || !empty($_GET) ? '<a id="top" href="' . $scripturl . '">' : '', empty($context['header_logo_url_html_safe']) ? $context['forum_name_html_safe'] : '<img src="' . $context['header_logo_url_html_safe'] . '" alt="' . $context['forum_name_html_safe'] . '">', !empty($context['current_action']) || !empty($_GET) ? '</a>' : '', '
```
Comment (SychO9) [inline Themes/default/index.template.php]: I don't believe it matters that much to make the edit here, in fact this change makes the style of the header link inconsistent where in the home page it looks different than in other pages, it should be reversed imho.

File: Sources/ManageBans.php (hunk 1 above)
Comment (dragomano) [reply Sources/ManageBans.php]: Done.

**Review by SychO9 — APPROVED** [review]
Looks good to me 🚀 !

Thank you @dragomano

**Review by XinYenFon — APPROVED** [review]
Looks good.

**Review by pouvik — APPROVED** [review]
changes work fine, do make the look feel good.  No errors that I can find.

**Comment by dragomano** [comment]
Is any chance to merge these changes before the Final?

**Comment by live627** [comment]
@dragomano can you please redo this? I can't pull it to resolve merge conflicts.

**Comment by dragomano** [comment]
I will do it in pieces.

**Pull request closed without merging**